stable/etcd-operator/templates/cluster.yaml

```diff
@@ -1,6 +1,6 @@
-{{- if .Values.cluster.enabled -}}
-apiVersion: "coreos.com/v1"
-kind: "EtcdCluster"
+{{- if and .Values.cluster.enabled (not .Release.IsInstall) -}}
```

Comment:
(query, no action required) is this TPR not required in install mode, when .Values.cluster.enabled is true?